GolfExample_TCAPI/scripts/common.js

```diff
@@ -26,5 +52,10 @@ GolfExample.getContext = function(parentActivityId) {
             id: parentActivityId
         };
     }
+    if (isAssessment){
+        ctx.contextActivities.grouping.push({
```

Comment:
Should this have been parent? isAssessment only applies to the questions of the assessment?